Stock.playground/Contents.swift

```diff
@@ -3,116 +3,147 @@ import Foundation
 
 struct StockGrant
 {
-    struct GrantDate
-    {
-        private(set) var foundationDate: NSDate
-
-        init (month: Int, day: Int, year: Int)
-        {
-            let signing = NSDateComponents.init()
-            signing.calendar = NSCalendar.currentCalendar()
-            signing.month = month
-            signing.day = day
-            signing.year = year
-            self.foundationDate = signing.date!
-        }
-    }
-
-
     let shares: UInt
     let strikePrice: Double
-    let grantDate: GrantDate
+    let grantDate: Date
     let cliffMonths: UInt
     let endMonths: UInt
 
-
-    // RSUs
-    init(shares: UInt, grantDate: GrantDate, cliffMonths: UInt, endMonths: UInt)
+    // RSU grants will use default value for strikePrice, NSO grants will set a strike price.
+    init(shares: UInt, grantDate: Date, cliffMonths: UInt, endMonths: UInt, strikePrice: Double = 0)
```

Comment:
ahhhh now I get it. didn't read the comment 😮.

Maybe better to `init(RSUs shares: Int` to drive the point home